flight/Modules/Actuator/actuator.c

```diff
@@ -338,7 +338,7 @@ static void actuator_task(void* parameters)
 float process_mixer(const int index, const float curve1, const float curve2,
 		ActuatorDesiredData *desired)
 {
-	int8_t *vector = get_mixer_vec(index);
+	typeof(&(*mixerSettings.Mixer1Vector)) vector = get_mixer_vec(index);
```

Comment:
consider for these typeof (mixerSettings.Mixer1Vector) (*vector) .  The dereferences below would need (*vector).  The upside is that this form (pointer to array) preserves type information so smart compilers (clang, some versions of gcc) can detect accesses past the end.

note that pointer to array and pointer to first element of an array have the same value, so the (*vector) in access is kinda.. vestigial.